modules/network/src/test/java/org/apache/ignite/network/internal/netty/NettyServerTest.java

```diff
@@ -35,6 +37,15 @@
  * Tests for {@link NettyServer}.
  */
 public class NettyServerTest {
+    /** Server. */
+    private NettyServer server;
+
+    /** */
+    @AfterEach
+    final void tearDown() {
```

Comment:
Why not public?

Reply:
Why should it be public? No one calls it anyway (except for JUnit)